x-pack/test/functional/config.base.js

```diff
@@ -500,6 +500,31 @@ export default async function ({ readConfigFile }) {
           },
         },
 
+        follower_index_user: {
```

Comment:
I'm not sure when https://github.com/elastic/kibana/issues/143720 will be looked into. Does it make sense to hold off on creating this new test user until it's being used?